lib/reporter/stats-reporter.js

```diff
@@ -0,0 +1,97 @@
+'use strict';
+
+const colors = require('colors');
+const filesize = require('filesize');
+const Logger = require('../logger');
+const percent = require('calc-percent');
+const program = require('commander');
```

Comment:
Don't use globals to pass values around. Rather pass `logLevel` or a `logger` instance as a parameter to this module.